composer.json

```diff
@@ -43,15 +55,5 @@
                 "jwhulette\\Pipes\\PipesServiceProvider"
```

Comment:
Please fix this or accept my pullrequest.

**jwhulette**\\Pipes\\PipesServiceProvider 
should be:
**Jwhulette**\\Pipes\\PipesServiceProvider